src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,125 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+import { getData } from '../../Api';
+
+export const FindMovie = ({ onMovies }) => {
+  const [value, setValue] = useState('');
+  const [errorFind, setError] = useState(false);
+  const [previewMovie, setMovie] = useState(null);
+
+  const getFilm = () => {
+    getData(value)
+      .then((data) => {
+        if (data.Response === 'False') {
+          return setError(true);
+        }
+
+        const {
+          Title,
+          Plot,
+          Poster,
+          imdbID,
+        } = data;
+
+        const newMovie = {
+          title: Title,
+          description: Plot,
+          imgUrl: Poster,
+          imdbUrl: `https://www.imdb.com/title/${imdbID}`,
+          imdbId: imdbID,
+        };
+
+        return setMovie(newMovie);
+      });
+  };
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+    setError(false);
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  const handlerSubmit = (event) => {
```

Comment:
```suggestion
  const handleSubmit = (event) => {
```
The name of the function should represent the action that this function does